openstack/sre/values.yaml

```diff
@@ -9,40 +9,42 @@ prometheus:
 
 # recording periods
 periods:
-  - 5m
-  - 30m
   - 1h
   - 2h
-  - 6h
   - 1d
-  - 3d
+  - 7d
 
 # SLO's
 slo:
-  api_error_ratio_percent:
-    arc-api: 0.1
-    barbican-api: 0.1
-    castellum-api: 0.1
-    cinder-api: 0.1
-    cfm: 0.1
-    designate-api: 0.1
-    elektra: 0.1
-    es-client: 0.1
-    es-manager: 0.1
-    glance: 0.1
-    grafana: 0.1
-    grafana-global: 0.1
-    hermes-api: 0.1
-    ironic-api: 0.1
-    keystone: 0.01
-    kibana: 0.1
-    limes-api-ccloud: 0.1
-    lyra: 0.1
-    maia: 0.1
-    maia-oprom: 0.1
-    manila-api: 0.1
-    neutron-server: 0.1
-    nova-api: 0.1
-    nova-console: 0.1
-    nova-placement-api: 0.1
-    octavia-api: 0.1
+  default_warning: 99.8
+  default_critical: 99.6
```

Comment:
Imho, putting the defaults here just as "default_warning" doesn't make sense if we ever want to support more than one metric, which having "api_availability_percent" as an extra dict implies. It needs to somehow align with supporting that, e.g.
```
slo:
  defaults:
    api_availability_percent:
      warning: 99.8
      critical: 99.6
```

That being said, it currently doesn't matter and could be done once we need more than one default or metric. Your choice.

Reply:
very true